c10/util/logging_is_not_google_glog.h

```diff
@@ -89,19 +89,21 @@ static_assert(
     "CAFFE2_LOG_THRESHOLD should at most be GLOG_FATAL.");
 // If n is under the compile time caffe log threshold, The _CAFFE_LOG(n)
 // should not generate anything in optimized code.
-#define LOG(n)                                 \
-  if (::c10::GLOG_##n >= CAFFE2_LOG_THRESHOLD) \
-  ::c10::MessageLogger((char*)__FILE__, __LINE__, ::c10::GLOG_##n).stream()
+#define LOG(n)                                                           \
+  if (::c10::GLOG_##n >= CAFFE2_LOG_THRESHOLD)                           \
+  ::c10::MessageLogger((const char*)__FILE__, __LINE__, ::c10::GLOG_##n) \
```

Comment:
Hmm, why any cast at all is needed here? `__FILE__` is a string literal, i.e. its by definition `const char *`, isn't it?
```suggestion
  ::c10::MessageLogger(__FILE__, __LINE__, ::c10::GLOG_##n) \
```